tools/telemetry/README.md

```diff
@@ -0,0 +1,22 @@
+### Telemetry tools
+
+Running the services specified in the `docker-compose.yml` file will give you access to:
+
+- The Jaeger UI with both the `/monitor` (latency, errors, req/sec) and `/search` (traces) tabs ready to use.
+- A Prometheus server used by Jaeger with enhanced monitoring data provided by OpenTelemetry.
+
+To get started:
+
+1. Start the necessary services by running `docker compose up` in this directory.
```

Comment:
I wonder if we should suggest running this using

```
docker-compose -f docker-compose.yml -f tools/telemetry/docker-compose.yml up -d
```

That way everything is on the same docker network. Is this setup collecting any data from mysql, redis, etc?

Reply:
> Is this setup collecting any data from mysql, redis, etc?

This depends entirely on what the Fleet server instruments, this setup just provides an endpoint to which Fleet can send the telemetry data (configured via `OTEL_EXPORTER_OTLP_ENDPOINT`).

Currently we instrument MySQL (by using a custom driver) and general information about requests (via a middleware).

The whole setup kind of looks like this:

```mermaid
graph LR
    subgraph Telemetry
        ot[OTEL Collector] --> p[Prometheus]
        ot[OTEL Collector] --> j[Jaeger]
        p[Prometheus] --> j[Jaeger]
    end

    subgraph Fleet
        m[(MySQL)] <--> ci((Fleet))
        r[(Redis)] <--> ci((Fleet))
        ci((Fleet)) --> ot[OTEL Collector]
    end
``` 



> I wonder if we should suggest running this using
> 
> ```
> docker-compose -f docker-compose.yml -f tools/telemetry/docker-compose.yml up -d
> ```

Thanks to your suggestions above, I got rid of `host.docker.internal`. With this in place I'd rather keep things separate, so you can "plug" this set of tools to a running setup just by restarting the server, what do you think?